app/src/main/java/com/codesoom/assignment/factories/ProductFactory.java

```diff
@@ -0,0 +1,75 @@
+package com.codesoom.assignment.factories;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductUpdateRequest;
+import org.springframework.util.StringUtils;
+
+/**
+ * 상품 생성 및 변경 담당
+ */
+public class ProductFactory {
+
+    private ProductFactory() {
+    }
+
+    /**
+     * 새로운 상품 반환
+     * @param name  상품명
+     * @param maker 메이커
+     * @param price 가격
+     * @param imagePath 이미지 경로
+     * @return 상품
+     */
+    public static Product getProduct(String name, String maker, Integer price, String imagePath) {
```

Comment:
이 메소드의 역할은 `get`이 아닙니다. 인스턴스를 생성하는 거죠.
```suggestion
    public static Product createNewProduct(String name, String maker, Integer price, String imagePath) {
```

Reply:
앗 넵 제가 이상하게 생각했네요 ㅠㅠ

Reply:
이상하다뇨 ㅎㅎ 그럴 수 있습니다. 상황에 따라 다 달라질 수 있는 내용이에요.